javascript/javascript3/week3/speedTime.js

```diff
@@ -0,0 +1,22 @@
+
+//When will we be there
+
+const travelInformation = {
+    speed: 50,//km/h
+    destinationDistance: 432,//km 
+  };
+
+  function notThisFunctionName(travelInformation){
```

Comment:
i think part of the assignment is for you to come up with a better function name ;)